docs/ha-scalable-wordpress.md

```diff
@@ -510,7 +525,333 @@ Follow this guide to set up a high availability (HA) and scalable WordPress depl
 
 - Enjoy!
 
+
+## Backup and Restore
+
+We need to backup both the database and the application files regularly and can restore the backups
+any time.
+
+There are many supported backends for the backup, we use Google Cloud Storage (GCS)
+in this example, however, you can apply the same with other kinds of backends.
+
+For backup, restore and disaster-recovery, we'll use:
```

Comment:
restoring

Reply:
restore is a noun, too

`(computing) The act of recovering data or a system from a backup.` from https://wikidiff.com/restoration/restore